python/sdss_access/path/path.py

```diff
@@ -716,3 +721,41 @@ def definitiondir(self, filetype, **kwargs):
 
 class AccessError(Exception):
     pass
+
+
+def _extract(template, example):
```

Comment:
Do you have any preferences on this being a separate standalone function or a `static method` attached to the `BasePath` class?

Reply:
+1 BasePath